sys/net/gnrc/netif/Kconfig

```diff
@@ -36,4 +36,13 @@ config GNRC_NETIF_MIN_WAIT_AFTER_SEND_US
         This value is expressed in microseconds. It is purely meant as a debugging
         feature to slow down a radios sending.
 
+config GNRC_NETIF_NONSTANDARD_6LO_MTU
+    bool "Enable usage of non standard MTU for 6LoWPAN network interfaces"
+    depends on MODULE_GNRC_NETIF_6LO
+    default n
```

Comment:
Sorry for spotting this late. Not need to default to `n`, that's the default behaviour.